property_groups/dream_prompt.py

```diff
@@ -166,12 +166,8 @@ def seed_clamp(self, ctx):
 }
 
 default_optimizations = Optimizations()
-if sys.platform == "darwin":
-    inferred_device = "mps"
-elif Pipeline.directml_available():
-    inferred_device = "privateuseone"
-else:
-    inferred_device = "cuda"
+inferred_device = Optimizations.infer_device()
```

Comment:
`inferred_device` is no longer needed here since optimization filtering was moved to `ui/panels/dream_texture.py`.
```suggestion
```